fontData.js

```diff
@@ -12,6 +12,76 @@ const buildAxes = font => {
   }));
 };
 
+const guessFontStyle = fontName => {
```

Comment:
What do you think of naming this `guessFontStyles` (plural), to make clear that it can return > 1 style.

Reply:
👍

Reply:
Thinking about this, the result is always a single style, e.g. "bold condensed". But I did rename it to `suggestFontStyle`, which sounds less magical ;-)